src/test/java/uk/nhs/hee/tis/revalidation/migration/processor/BatchDataProcessorTest.java

```diff
@@ -54,6 +57,9 @@
   @InjectMocks
   BatchDataProcessor batchDataProcessor;
 
+  @Mock
+  GmcHexStringConverter gmcHexStringConverter;
+
```

Comment:
Given the simplicity of the converter we should avoid mocking it, replacing this mock with a spy would allow the inject to still work.

```
@Spy
GmcHexStringConverter gmcHexStringConverter = new GmcHexStringConverter();
```